frontend/src/views/camp/Camp.vue

```diff
@@ -3,17 +3,25 @@ Displays several tabs with details on a single camp.
 -->
 
 <template>
-  <v-container fluid>
+  <v-container fluid :style="layoutStyle">
     <router-view v-bind="$attrs" />
   </v-container>
 </template>
 
 <script>
 export default {
-  name: 'Camp'
+  name: 'Camp',
+  props: {
+    layout: { type: String, required: false, default: 'true' }
```

Comment:
Wieso default-Wert 'true'?

Reply:
Das ist falsch, da sollte full sein.
```suggestion
    layout: { type: String, required: false, default: 'full' }
```